test/hotspot/gtest/logging/test_asynclog.cpp

```diff
@@ -244,67 +244,68 @@ TEST_VM_F(AsyncLogTest, logBuffer) {
 }
 
 TEST_VM_F(AsyncLogTest, droppingMessage) {
+  if (AsyncLogWriter::instance() == nullptr)
+    return;
```

Comment:
Please use braces, even for single statement if-bodies.

Reply:
thanks. I resolve it.